docs/guides/first-app.mdx

```diff
@@ -2,30 +2,29 @@
 
 import TabItem from '@theme/TabItem';
 import Tabs from '@theme/Tabs';
+import CodeBlock from '@theme/CodeBlock';
+import {SampleConfigProvider, SampleCodeBlock, SampleConfigEditor} from '@site/src/components/SampleConfigProvider';
+
+<SampleConfigProvider defaultTenant="blog">
 
 ## Introduction
 
-This guide walks through the steps required to deeply integrate an application with SpiceDB.
+This guide walks through the steps required to deeply integrate an application with Authzed or SpiceDB.
 Not all software requires this level of integration, but it is preferable for greenfield applications or applications that are central in an architecture with multiple services.
 
 Instead of introducing an unfamiliar example app and altering various locations in its code, this guide is written such that each step is a standalone snippet of code that demonstrates an integration point and finding where those points exist in your codebase is an exercise left to the reader.
 
 ## Prerequisites
 
-- An [Authzed] permission system **OR** a running instance of [SpiceDB][SpiceDB]
-- An [API Token] with `admin` access **OR** the configured preshared key for SpiceDB
+One of:
+- An [Authzed] permission system and associated [API Token] with `admin` access
+- A running instance of [SpiceDB][SpiceDB] with the configured preshared key for SpiceDB
 
 [Authzed]: https://app.authzed.com
 [API Token]: /reference/api#authentication
 [SpiceDB]: https://github.com/authzed/spicedb
 
-:::warning
-Throughout this guide various substitutions must be made in the provided code snippets:
-
-- Authzed users must replace the `blog` prefix throughout this guide with their own permission system's slug
-- SpiceDB users must replace `grpc.authzed.com:443` with their own SpiceDB endpoint
-- All users must replace `t_your_token_here_1234567deadbeef` with their API token or preshared key
-:::
+<SampleConfigEditor/>
```

Comment:
Should one of these be default selected to make this just 2 states instead of 3?

Reply:
I explicitly wanted it to be a tristate, with it defaulting otherwise. It ensures that someone following along *has* to click (if they want the auto-replace to use non-defaults), which its a more meaningful action and thus makes it more likely they've take the prerequisite steps. Its subtle, but is a general strategy to help guide people